packages/atomic/src/components/atomic-smart-snippet/atomic-smart-snippet.tsx

```diff
@@ -227,7 +182,15 @@ export class AtomicSmartSnippet implements InitializableComponent {
           {this.renderQuestion()}
           {this.renderContent()}
           <footer part="footer">
-            {this.renderSource()}
+            {this.smartSnippetState.source ? (
+              <atomic-smart-snippet-source
+                source={this.smartSnippetState.source}
+                smartSnippet={this.smartSnippet}
+                isSuggestion={false}
```

Comment:
Parts must be exported here, otherwise they won't be customizable from outside `atomic-smart-snippet`:
```suggestion
                isSuggestion={false}
                exportparts="source-url,source-title"
```

Reply:
Even without shadow dom?

Reply:
ah, if there's no ShadowDOM, I don't think it's necessary